Headers are not installed in libdir during compilation.
Search them in the build directory.
Index: oyranos-0.4.0/examples/image_display/oyranos_display_helpers.c
===================================================================
--- oyranos-0.4.0.orig/examples/image_display/oyranos_display_helpers.c	2012-04-15 12:25:37.834465263 +0200
+++ oyranos-0.4.0/examples/image_display/oyranos_display_helpers.c	2012-04-15 12:26:34.998463564 +0200
@@ -13,7 +13,7 @@
  */
 
 #include <oyranos.h>
-#include <alpha/oyranos_alpha.h>
+#include "oyranos_alpha.h"
 #include "oyranos_display_helpers.h"
 
 int oy_display_verbose = 0;
Index: oyranos-0.4.0/examples/libraw/oyranos_file.cpp
===================================================================
--- oyranos-0.4.0.orig/examples/libraw/oyranos_file.cpp	2012-04-15 12:49:26.522422764 +0200
+++ oyranos-0.4.0/examples/libraw/oyranos_file.cpp	2012-04-15 12:49:43.638422256 +0200
@@ -15,7 +15,7 @@
  */
 /* c++ -Wall -g oyranos_file.cpp `pkg-config --cflags --libs oyranos` -o oyranos-file */
 
-#include <alpha/oyranos_alpha.h>
+#include "oyranos_alpha.h"
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
